services/tasks-svc/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"common"
+	pb "gen/proto/services/tasks_svc/v1"
+	daprd "github.com/dapr/go-sdk/service/grpc"
+	"hwes/eventstoredb"
+	"tasks-svc/internal/task/api"
+	"tasks-svc/internal/task/service"
+)
+
+const ServiceName = "tasks-svc"
+
+// Version is set at compile time
+var Version string
+
+func main() {
+	common.Setup(ServiceName, Version, true)
+
+	eventStore := eventstoredb.SetupEventStoreByEnv()
+	aggregateStore := eventstoredb.NewAggregateStore(eventStore)
```

Comment:
I think there should be only one line for this. If I only need the eventStore to create the aggregateStore, why not just create the aggregateStore?

Reply:
Later on we need to share the eventStore instance for projections.